com.unity.render-pipelines.core/Editor/AssemblyInfo.cs

```diff
@@ -0,0 +1,5 @@
+using System.Runtime.CompilerServices;
+
+// remove as needed when/if MaterialUpgrader.RenameType can be made public
+[assembly: InternalsVisibleTo("Unity.RenderPipelines.Universal.Editor")]
+[assembly: InternalsVisibleTo("Unity.Testing.SRP.Universal-Upgrade.Editor")]
```

Comment:
Could this be remove now since `RenameType` and `NewShader` are public?

Reply:
indeed! Removed in c12129437be685e5764c53f50ec86f1ead708bec